backend/src/main/java/atlantbh/restaurants/services/UserService.java

```diff
@@ -0,0 +1,46 @@
+package atlantbh.restaurants.services;
+
+import atlantbh.restaurants.controllers.dto.UserRegistrationDTO;
+import atlantbh.restaurants.models.Location;
+import atlantbh.restaurants.models.User;
+import atlantbh.restaurants.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import javax.swing.text.html.Option;
+import javax.validation.Valid;
+import java.util.Optional;
+
+@Service
+public class UserService extends BaseService<User, UserRepository> {
+
+    @Autowired
+    LocationService locationService;
+
+    public User findByEmail(String email) {
+        return repo.findByEmail(email);
```

Comment:
call it repository

Reply:
Fixed in 92c4f7264b274d7da369717e0ed1478a93d30ee4